simple/Cargo.toml

```diff
@@ -14,6 +14,7 @@ encoding_rs = { workspace = true }
 
 [features]
 cublas = ["llama-cpp-2/cublas"]
+metal =  ["llama-cpp-2/metal"]
```

Comment:
I think this should be default.

Reply:
the feature `metal` is hidded by `llama-cpp-2`, and `simple` could turn on/off it manually.